addons/event/models/res_partner.py

```diff
@@ -9,6 +9,7 @@ class ResPartner(models.Model):
     event_count = fields.Integer("Events", compute='_compute_event_count', help="Number of events the partner has participated.")
 
     def _compute_event_count(self):
```

Comment:
why not this instead?
```suggestion
     def _compute_event_count(self):
        if not self.user_has_groups('event.group_event_user'):
            self.event_count = 0
            return
```

Reply:
We can but its been like this as other places too
https://github.com/odoo/odoo/pull/39446/files

Not sure if adding  `self.event_count = 0` makes major impact

Reply:
It doesn't change much in performance (thanks to the recent orm changes), and we find it clearer to place the default value assignation at the beginning of the compute.